<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectPositionWidget</class>
 <widget class="QDialog" name="ObjectPositionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_WidgetPosition</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>560</width>
        <height>533</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>坐标参考及格式</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="3">
           <widget class="QComboBox" name="cbTimeFormat">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::NoInsert</enum>
            </property>
            <item>
             <property name="text">
              <string>yyyy-MM-dd hh:mm:ss</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>double</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="sRefLabel_3">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>坐标格式</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QComboBox" name="cbSpatialFormat">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>X Y Z</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>L B H</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>X Y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>L B</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="trLabel">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>时间格式</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="sRefLabel_2">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>时间参考</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbTimeRef">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="sRefLabel">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>空间参考</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cbSpatialRef">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="4">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
         <zorder>cbSpatialRef</zorder>
         <zorder>sRefLabel</zorder>
         <zorder>cbTimeRef</zorder>
         <zorder>sRefLabel_2</zorder>
         <zorder>trLabel</zorder>
         <zorder>cbTimeFormat</zorder>
         <zorder>sRefLabel_3</zorder>
         <zorder>cbSpatialFormat</zorder>
         <zorder>horizontalSpacer_3</zorder>
         <zorder>horizontalSpacer_4</zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>坐标设置</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableWidget" name="PostionTable">
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>时间值</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>L</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>B</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>H</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>rx</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>ry</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>rz</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="PathImportBtn">
              <property name="text">
               <string>导入路径</string>
              </property>
              <property name="icon">
               <iconset resource="../ModelingTool.qrc">
                <normaloff>:/ModelingTool/Images/PostionPathImport.ico</normaloff>:/ModelingTool/Images/PostionPathImport.ico</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PointEmptyBtn">
              <property name="text">
               <string>清空点</string>
              </property>
              <property name="icon">
               <iconset resource="../ModelingTool.qrc">
                <normaloff>:/ModelingTool/Images/PostionPathEmpty.png</normaloff>:/ModelingTool/Images/PostionPathEmpty.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PointAddBtn">
              <property name="text">
               <string>增加点</string>
              </property>
              <property name="icon">
               <iconset resource="../ModelingTool.qrc">
                <normaloff>:/ModelingTool/Images/PostionPointAdd.png</normaloff>:/ModelingTool/Images/PostionPointAdd.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PointDeleteBtn">
              <property name="text">
               <string>删除点</string>
              </property>
              <property name="icon">
               <iconset resource="../ModelingTool.qrc">
                <normaloff>:/ModelingTool/Images/PostionPointRemove.png</normaloff>:/ModelingTool/Images/PostionPointRemove.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>其它</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>该功能为保留，用于设置以轨道参数计算位置和关联型动态计算位置的对象</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QRadioButton" name="orbitRadio">
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>轨道</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="dynamicPosRadio">
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>关联</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="OrbitBtn">
              <property name="text">
               <string>轨道参数</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="DynamicPosBtn">
              <property name="text">
               <string>关联参数</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../ModelingTool.qrc"/>
 </resources>
 <connections/>
</ui>
